docs/versions.json

```diff
@@ -1,5 +1,9 @@
 [
+  "v0.19.3",
```

Comment:
Is the expected workflow to flip this on before it is a stable release?

Reply:
Will be updated once the [setStable.js](https://github.com/noir-lang/noir/blob/master/docs/scripts/setStable.js) script runs, so should be fine.